app/views/common/_pagination.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="form-group">
+  <%= select_tag('per_page',
+                 options_for_select([5, 10, 15, 25, 50], params[:per_page] || Setting[:entries_per_page].to_i),
+                 label: _('per page'),
+                 onChange: "$(this).parentsUntil('.form').submit()",
```

Comment:
please extract it to a function under tfm.tools, this will allow us to add other actions (such as spinner or other events).